xml-csharp/SelectingXmlNodesWithXpath/SelectingXmlNodesWithXpathTests/SelectingXmlNodesWithXpathTest.cs

```diff
@@ -0,0 +1,99 @@
+using System.Xml;
+
+namespace SelectingXmlNodesWithXpathTests;
+
+public class SelectingXmlNodesWithXpathTest
+{
+    private XmlDocument Document { get; set; }
+    private Dictionary<string, string> ExpectedResults { get; set; }
+
+    public SelectingXmlNodesWithXpathTest()
+    {
+        Document = new XmlDocument();
+        Document.Load("BooksCatalog.xml");
+
+        ExpectedResults = new Dictionary<string, string>()
+        {
+            {
+                "Book1",
+                """<book id="1">""" +
+                    """<author>King, Stephen</author>""" +
+                    """<title>IT</title>""" +
+                    """<genre>Horror</genre>""" +
+                    """<price>40.00</price>""" +
+                """</book>"""

```

Comment:
Please review the documentation for raw string literals, there is no need for the `+` in this, we can just write the string in a formatted way, including new lines and have only one opening and closing set of `"""`
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/raw-string

Reply:
@jshergal, the string was split like that because the `OuterXml` property is not formatted. I followed your suggestion, but, to that, I had to add a new method to format the strings. In the end, I think that the final result is better.

I also don't know why I was printing the result as an unformatted XML. I'll update the article with this new printing format.